src/library/trackset/setlogfeature.cpp

```diff
@@ -43,8 +43,18 @@ SetlogFeature::SetlogFeature(
     // remove unneeded entries
     deleteAllUnlockedPlaylistsWithFewerTracks();
 
-    // Create empty placeholder playlist for YEAR items
     QString placeholderName = "historyPlaceholder";
+    // remove previously created placeholder playlists
+    const auto pls = m_playlistDao.getPlaylists(PlaylistDAO::PLHT_UNKNOWN);
+    QStringList plsToDelete;
+    for (const auto& pl : pls) {
+        if (pl.second.startsWith(placeholderName)) {
```

Comment:
auto above and second here is not well readable. 
Not your fault. If you have the mood to improve it go ahead. 
Else we can ignore it in this bufix PR

Reply:
Yeah, auto was simply faster to type.
I'll change it.